kratos/tests/auxiliar_files_for_python_unittest/vtk_output_process_ref_files/asciiQuadraticHexahedra3D27N/Main_0_2.vtk

```diff
@@ -0,0 +1,156 @@
+# vtk DataFile Version 4.0
+vtk output
+ASCII
+DATASET UNSTRUCTURED_GRID
+POINTS 27 float
+0.00000000e+00 0.00000000e+00 0.00000000e+00 
```

Comment:
You can reduce size of the file removing the 000000

Reply:
done